modules/rgbd/src/tsdf.cpp

```diff
@@ -859,14 +859,13 @@ static cv::UMat preCalculationPixNormGPU(int depth_rows, int depth_cols, Vec2f f
         throw std::runtime_error("Failed to create kernel: " + errorStr);
 
     AccessFlag af = ACCESS_READ;
-    UMat pixNorm = _pixNorm.getUMat(af);
```

Comment:
Agreed, `getUMat()` is local only wrapper - it should not be "returned" or stored somewhere.